frontend/packages/client/src/components/FlipsContainer/FlipCardFooter.js

```diff
@@ -0,0 +1,88 @@
+import { Svg } from '@cast/shared-components';
+import { FilterValues } from 'const';
+import { parseDateFromServer } from 'utils';
+
+const STATUS_BUTTONS = {
+  [FilterValues.active]: (
+    <button
+      class={`button is-rounded is-small has-text-weight-semibold has-background-orange has-border-orange`}
+    >
+      Cast Your Vote
+    </button>
+  ),
+  [FilterValues.pending]: (
+    <button
+      class={`button is-rounded is-small has-text-weight-semibold has-background-orange has-border-orange`}
+    >
+      Upcoming...
+    </button>
+  ),
+  [FilterValues.closed]: (
+    <button
+      class={`button is-rounded is-small has-text-weight-semibold has-background-green has-border-green`}
```

Comment:
Please lets use the same styling as in the community page for the buttons for consistency, for completed Flips its `has-background-success`

Reply:
The color of complete button I've made consistent with the website green theme.
In community page, a new dark green color is breaking the flow. Attaching images for reference.
Please let me know if still we should change that to dark green.
![Screenshot 2023-08-18 at 12 08 59 PM](https://github.com/onflow/CAST/assets/135014949/39406ea5-c3de-4224-a3a7-7f8decebae86)

Reply:
Need to add the tick icon after complete

Reply:
@AmarildoGrembi I've added button for active status and badges for other statuses.
Please check and let me know if we can improve anything.

Reply:
Thats a great idea actually, thanks! One last thing, once a user casts a vote - could we please show the badge + days remaining for the proposal to be completed, similar to how its done in the community page?